kong/api/endpoints.lua

```diff
@@ -396,7 +396,7 @@ end
 --
 -- /services/:services
 -- /services/:services/routes/:routes
-local function get_entity_endpoint(schema, foreign_schema, foreign_field_name, method)
+local function get_entity_endpoint(schema, foreign_schema, foreign_field_name, method, inversed)
```

Comment:
`inversed` is not a standard English word — it should be either `inverted` or `reversed`

Reply:
i renamed the variable to `is_foreign_entity_endpoint`, doesn't need further explanation in my opinion.